packages/web-components/src/slider/slider.spec.md

```diff
@@ -76,6 +76,7 @@ None
   - `rail` is in Fluent React; `track` is in FAST Foundation
   - No value indicators to the left/right of slider in FAST Slider. These should be present in Fluent version. This will be handled through slotting in the the SliderLabel component.
   - No default support for the ticks on the slider when `step` is set. This will be handled through slotting in the SliderLabel component.
+  - Added attribute of `marks` whose default value is false. This will show/hide the ticks on the slider when `step` is set.
```

Comment:
Where does the language "marks" come from? is this consistent with fluent design language?